src/Jss.js

```diff
@@ -15,6 +15,11 @@ export default class Jss {
     this.sheets = new SheetsRegistry()
     this.plugins = new PluginsRegistry()
     this.uid = uid
+    if (__VERSION__) {
```

Comment:
This `if` is optimisation for webpack. To not contain all data of the `package.json` in `dist` file, we need to add this line.